<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>summary</class>
 <widget class="QDialog" name="summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>812</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>812</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>812</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>375</width>
     <height>812</height>
    </size>
   </property>
   <widget class="QLabel" name="summary_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>375</width>
      <height>812</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>812</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/summary.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="go_secpage">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>50</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px; background-color:rgba(255,255,255,0.0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="year_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>0</height>
     </size>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p &gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 14pt &quot;맑은 고딕&quot;;
	color: rgb(0,0,0);
	background-color: white;
	border: 0px solid rgb(0,0,0);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
border: 2px solid rgb(0,0,0);
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="month_btn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p &gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 14pt &quot;맑은 고딕&quot;;
	color: rgb(0,0,0);
	background-color: white;
	border: 0px solid rgb(0,0,0);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Month</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="week_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p &gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	font: 700 14pt &quot;맑은 고딕&quot;;
	color: rgb(0,0,0);
	background-color: white;
	border: 0px solid rgb(0,0,0);
	border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>Week</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>360</y>
      <width>131</width>
      <height>441</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>113</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>360</y>
      <width>113</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:none;</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>381</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255,255,255,1.0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;맑은 고딕&quot;;
background-color: rgba(255, 255, 255, 0.0);
color: rgb(255,255,255);
border: 0px;</string>
    </property>
    <property name="text">
     <string>Total balance</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="money_line">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 40pt &quot;맑은 고딕&quot;;
background-color: rgba(255, 255, 255, 0.0);
color: rgb(255,255,255);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="date_info">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>120</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;맑은 고딕&quot;;
background-color: rgba(255, 255, 255, 0.0);
color: rgb(255,255,255);
border: 0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>lineEdit_3</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>summary_pic</zorder>
   <zorder>go_secpage</zorder>
   <zorder>label</zorder>
   <zorder>year_btn</zorder>
   <zorder>month_btn</zorder>
   <zorder>week_btn</zorder>
   <zorder>line</zorder>
   <zorder>money_line</zorder>
   <zorder>label_2</zorder>
   <zorder>date_info</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
